src/System.Device.Ports/System.Device.Ports.SerialPort/SerialPort.Settings.cs

```diff
@@ -65,12 +86,16 @@ public partial class SerialPort
         /// The name of the serial port whose default value is platform dependent
         /// and set to a proper default name in the derived platform-specific classes
         /// </summary>
-        protected string _portName = String.Empty;
+        protected string _portName = string.Empty;
```

Comment:
protected members should follow public API rules:
- `PortName`
- property rather than field

Reply:
and could be public IMHO

Reply:
There is already a public property named `PortName`.
The reason for the field being protected is because the default name for the Windows and Linux implementation is different.
In this case I can use the public setter of the property (I am doing it). But there are other fields that need to be accessed directly from the platform-specific implementation to bypass the validation (see `_rtsEnable`).